packages/leap/lib/src/leap_map.dart

```diff
@@ -59,10 +59,15 @@ class LeapMap extends PositionComponent with HasGameRef<LeapGame> {
     }
   }
 
-  static Future<LeapMap> load(String tiledMapPath, double tileSize) async {
+  static Future<LeapMap> load({
+    required double tileSize,
+    required String tiledMapPath,
+    String prefix = '',
```

Comment:
This introduces a bug and breaks at runtime. The default value needs to match `TiledComponent.load`:

```
  String prefix = 'assets/tiles/',
```